.github/comment_templates/connector_dependency_template.md

```diff
@@ -1,25 +1,54 @@
 <!--- this comment is for `report-connectors-dependency.yml` identification, do not remove -->
 
-NOTE ⚠️ Changes in this PR affect the following connectors. Make sure to run corresponding integration tests:
+NOTE ⚠️ Changes in this PR affect the following connectors. Make sure to do the following as needed:
+- Run integration tests
+- Bump connector version
+- Add changelog
+- Publish the new version
 
-<details>
+<details {source_open}>
 <summary>
-Sources ({num_sources})
+
+### {source_status_summary} Sources ({num_sources})
 
 </summary>
 
-{sources}
+| Connector | Version | Changelog | Publish |
+| --- | :---: | :---: | :---: |
+{source_rows}
 
 </details>
 
-<details>
+<details {destination_open}>
 <summary>
-Destinations ({num_destinations})
+
+### {destination_status_summary} Destinations ({num_destinations})
 
 </summary>
 
-{destinations}
+| Connector | Version | Changelog | Publish |
+| --- | :---: | :---: | :---: |
+{destination_rows}
 
 </details>
 
 {others}
+
+<details>
+
+<summary>
+
+### Notes
```

Comment:
Suggestion: rename to something more explicit, e.g. "Status reference" 

Upon first glance I totally missed that this was a dropdown and not just an empty notes section. Your demo made it clear, but this might help people who haven't seen it.

That being said, this section is super helpful!

Reply:
Good point. Will do in a follow-up PR.